Add images to the Block step page (#2040)
buildkite/docs: pages/pipelines/block_step.md

```yml
steps:
  - block: "\:rocket\: Are we ready?"
```
{: codeblock-file="pipeline.yml"}

> <%= image "block-step-button.png", alt: "Screenshot of button to press to unblock a block step" %>

Reviewer: As mentioned offline, I was thinking that we might need the widths + heights for these images but now that I see them full width I think it's probably OK – assuming the image is at least double the container column size.

Author: Yeah they are exported at 2x 1080

> The command step added in the above example will upload the trigger step and add it to the end of our pipeline at runtime.

Reviewer 2: A style fix while we're here:

```suggestion
The command step added in the above example uploads the trigger step and adds it to the end of the pipeline at runtime.
```

> <%= image "block-trigger-pipeline.png", width: 2328/2, height: 952/2, alt: "Screenshot of pipeline showing the uploaded trigger step" %>

Reviewer 2: Also remove the corresponding file from the repo.

> In the pipeline you're triggering, you will be able to use the meta-data that you have passed through as if it was set during the triggered build.

Reviewer 2: Another style fix while we're here:

```suggestion
In the pipeline you're triggering, you can use the meta-data passed through as if it was set during the triggered build.
```
